pkg/ebpf/controlplane/controller.go

```diff
@@ -0,0 +1,161 @@
+package controlplane
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aquasecurity/libbpfgo"
+
+	"github.com/aquasecurity/tracee/pkg/capabilities"
+	"github.com/aquasecurity/tracee/pkg/containers"
+	"github.com/aquasecurity/tracee/pkg/ebpf/probes"
+	"github.com/aquasecurity/tracee/pkg/errfmt"
+	"github.com/aquasecurity/tracee/pkg/events"
+	"github.com/aquasecurity/tracee/pkg/events/parse"
+	"github.com/aquasecurity/tracee/pkg/logger"
+	"github.com/aquasecurity/tracee/types/trace"
+)
+
+const pollTimeout int = 300 // from tracee.go (move to a consts package?)
+
+// Control Plane probe handles
+const (
+	cgroupMkdirControlProbe probes.Handle = iota
+	cgroupRmdirControlProbe
+)
+
+type Controller struct {
+	ctx            context.Context
+	signalChan     chan []byte
+	lostSignalChan chan uint64
+	bpfModule      *libbpfgo.Module
+	probeGroup     *probes.ProbeGroup
+	signalBuffer   *libbpfgo.PerfBuffer
+	cgroupManager  *containers.Containers
+}
+
+func NewController(bpfModule *libbpfgo.Module, cgroupManager *containers.Containers) (*Controller, error) {
+	var err error
+	p := &Controller{
+		signalChan:     make(chan []byte, 100),
+		lostSignalChan: make(chan uint64),
+		bpfModule:      bpfModule,
+		cgroupManager:  cgroupManager,
+	}
+	p.signalBuffer, err = bpfModule.InitPerfBuf("signals", p.signalChan, p.lostSignalChan, 1024)
+	if err != nil {
+		return nil, err
+	}
+
+	p.probeGroup = probes.NewProbeGroup(bpfModule, map[probes.Handle]probes.Probe{
+		cgroupMkdirControlProbe: probes.NewTraceProbe(probes.RawTracepoint, "cgroup:cgroup_mkdir", "cgroup_mkdir_signal"),
```

Comment:
Could you please format `NewTraceProbe(<enter>...,<enter>)` to keep it less than 100 cols ? We're going to start enforcing it soon (ok if you don't it know, just bringing awareness).